### ISSUE TITLE
Tomcat8 cartridge

### ISSUE THREAD

**Issue by beradrian** [worldline/openshift-cartridge-tomcat#3]
Even though Tomcat8 is present for quite some time, there's no cartridge for it.

**Comment by SimonEast** [comment]
Yes, I'm also looking for a Tomcat 8 cartridge, but none seems to be around yet. :-(
